Formula/z80dasm.rb

```diff
@@ -5,6 +5,11 @@ class Z80dasm < Formula
   sha256 "76d3967bb028f380a0c4db704a894c2aa939951faa5c5630b3355c327c0bd360"
   license "GPL-2.0-or-later"
 
+  livecheck do
+    url "https://www.tablix.org/~avian/z80dasm"
+    regex(/href=.*?z80dasm[._-]v?(\d+(?:\.\d+)+[a-z]?)\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?z80dasm[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

I don't see any archive files that have a trailing character in the filename (e.g., `example-1.2.3e`), so we can stick with the standard format for matching in tarballs like: `/href=.*?example[._-]v?(\d+(?:\.\d+)+)\.t/i`.

Reply:
What you mention makes total sense. This was me doing some copy pasta from another formula :p let me update the regex then.